lib/batch/institutional_nucats_template.rb

```diff
@@ -0,0 +1,40 @@
+# https://github.com/galterlibrary/digital-repository/issues/691
+# Create 10 'template' institutional collections, adding another 3 template
+# collections to each. For more immediate use when needed.
+
+# NUCATS Grants Repository
+parent_nucats_grants = Collection.find("f2bf6e1d-0e32-4ce2-a52e-bb0522d5708d")
+
+10.times do |i|
+  template = Collection.new(
+    title: "template-#{i}", 
+    tag: ["grant proposals", "research support", "peer-reviewed grants"]
+  )
+  template.apply_depositor_metadata(parent_nucats_grants.depositor)
+  template.save!
```

Comment:
Do you need to `save!` here or can you get away with only saving later (at the end of the parent loop)?